package.json

```diff
@@ -4,59 +4,59 @@
   "private": true,
   "dependencies": {
     "@databiosphere/bard-client": "^0.1.0",
-    "@fortawesome/fontawesome-svg-core": "^1.2.32",
-    "@fortawesome/free-regular-svg-icons": "^5.15.1",
-    "@fortawesome/free-solid-svg-icons": "^5.15.1",
-    "@fortawesome/react-fontawesome": "^0.1.11",
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
+    "@fortawesome/free-regular-svg-icons": "^5.15.3",
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
+    "@fortawesome/react-fontawesome": "^0.1.14",
     "animate.css": "^4.1.1",
     "array-move": "^3.0.1",
-    "clipboard-polyfill": "^3.0.1",
-    "color": "^3.1.2",
-    "date-fns": "^2.16.1",
+    "clipboard-polyfill": "^3.0.3",
+    "color": "^3.1.3",
+    "date-fns": "^2.21.1",
     "debounce-promise": "^3.1.2",
-    "downshift": "^6.0.6",
-    "file-saver": "^2.0.2",
-    "filesize": "^6.1.0",
+    "downshift": "^6.1.3",
+    "file-saver": "^2.0.5",
+    "filesize": "^6.3.0",
     "github-markdown-css": "^4.0.0",
     "history": "^4.10.1",
-    "iframe-resizer": "^4.2.11",
+    "iframe-resizer": "^4.3.2",
     "igv": "2.3.5",
-    "jszip": "^3.5.0",
-    "lodash": "^4.17.20",
-    "marked": "^0.7.0",
-    "outdated-browser-rework": "^2.10.0",
+    "jszip": "^3.6.0",
+    "lodash": "^4.17.21",
+    "marked": "^2.0.3",
+    "outdated-browser-rework": "^3.0.1",
     "path-to-regexp": "^5.0.0",
     "prismjs": "^1.23.0",
     "prop-types": "^15.7.2",
-    "qs": "^6.9.4",
-    "react": "~16.13.0",
-    "react-collapse": "^5.0.1",
-    "react-dom": "npm:@hot-loader/react-dom@~16.13.0",
+    "qs": "^6.10.1",
+    "react": "17.0.1",
+    "react-collapse": "^5.1.0",
+    "react-dom": "npm:@hot-loader/react-dom@17.0.1",
     "react-draggable": "^4.4.3",
-    "react-dropzone": "^11.2.0",
-    "react-favicon": "^0.0.18",
-    "react-focus-lock": "^2.4.1",
+    "react-dropzone": "^11.3.2",
+    "react-favicon": "^0.0.21",
```

Comment:
@Shakespeared and i found that the BioDataCatalyst favicon is not showing up anymore (checked on dev branch locally and it works there)

Reply:
yep works now thanks!